### PR TITLE
Compare unknown order

### LINKED ISSUE #649: Change order of compare report
first card should be incidence, second be unknowns map

### PR DESCRIPTION
**Pull request by kkatzen**
Closes #649
I also noticed that the compare reports are getting mis-aligned if certain metrics are missing so i adjusted that. See screenshots:
Screen Shot 2021-04-22 at 2 13 04 PM
Screen Shot 2021-04-22 at 2 13 11 PM
Screen Shot 2021-04-22 at 2 13 18 PM
Screen Shot 2021-04-22 at 2 13 32 PM

### CODE REVIEW

File: frontend/src/reports/TwoVariableReport.tsx [hunk 1]
```diff
@@ -168,47 +171,93 @@ function TwoVariableReport(props: {
...
+              )}
+            />
           </>
         )
       )}
     </Grid>
   );
 }
 
+function RowOfOptionalMetrics(props: {
```
Comment (vanshkumar) [inline frontend/src/reports/TwoVariableReport.tsx:215]: Would a better name for this be RowOfTwoOptionalMetrics to make it clear it's only for two variables?

**Review by vanshkumar — APPROVED** [review]

**Comment by vanshkumar** [comment]
Also FYI that pct_share will be added in for poverty very soon (in PR) so the missing metrics alignment issue shouldn't be a problem after that

File: frontend/src/reports/TwoVariableReport.tsx (hunk 1 above)
Comment (kkatzen) [reply frontend/src/reports/TwoVariableReport.tsx:215]: ya good idea

**Comment by kkatzen** [comment]
Ya I did realize that poverty pct_share would be added soon but I figure this probably won't be the last time some weird edge resurfaces this issue I might as well fix it

**Pull request merged**